.ci/Jenkinsfile

```diff
@@ -51,7 +52,7 @@ pipeline {
       options { skipDefaultCheckout() }
       steps {
         cleanup()
-        withGoEnv(){
+        withGoEnv(version: "${GO_VERSION}"){
           dir("${BASE_DIR}"){
             sh(label: 'local',script: 'make local')
           }
```

Comment:
```suggestion
        dir("${BASE_DIR}"){
          withGoEnv(version: "${GO_VERSION}"){
            sh(label: 'local',script: 'make local')
          }
```

Reply:
Fixed in https://github.com/elastic/fleet-server/pull/346/commits/21328bf142b9578aa8ca73b4c912c0259cf03aa1